Dockerfile

```diff
@@ -9,17 +9,15 @@ ENV log_dir ${home_dir}/logs
 ENV deploy_dir ${home_dir}/deploy
 
 # Install common tools
-RUN set -x \
-    && apk update && apk upgrade \
-    && apk add --no-cache bash \
-    && apk add --no-cache fontconfig \
-    && apk add --no-cache ttf-dejavu 
+RUN set -x \ 
+    && yum update && yum upgrade \
+    && yum install bash fontconfig 
 
 # Add scripts
 ADD scripts/bootstrap.sh /
 
 # Create directories
-RUN mkdir -p ${log_dir} \
-    && mkdir -p ${deploy_dir}
+RUN mkdir -p ${log_dir}
```

Comment:
could be a single `RUN` command like before to avoid creating an extra image layer

Reply:
Dont't know why did I change this, but definitely should use only one `RUN` command